updating/preparing-eus-eus-upgrade.adoc

```diff
@@ -0,0 +1,22 @@
+[id="preparing-eus-eus-upgrade"]
+= Performing to perform an EUS to EUS upgrade
+include::modules/common-attributes.adoc[]
+:context: eus-to-eus-upgrade
+
+toc::[]
+
+Due to fundamental Kubernetes design, all {product-title} upgrades between minor versions must be serialized.
+You must upgrade from 4.8 to 4.9 and then to 4.10. You cannot upgrade from 4.8 to 4.10 directly.
+However, beginning with the upgrade from 4.8 to 4.9 to 4.10, administrators who wish to upgrade between two Extended Update Support (EUS) versions can do so incurring only a single reboot of non-master hosts.
+
+There are a number of caveats to consider when attempting an EUS to EUS upgrade.
+
+* EUS to EUS upgrades is only offered after upgrades between all versions involved have been made available in `stable` channels.
+* If you encounter issues during or after upgrading to the odd-numbered minor version but before upgrading to the next even-numbered version, then remediation of those issues may require that non-master hosts complete the upgrade to the odd-numbered version before moving forward.
+* You can complete the upgrade process during multiple maintenance windows by pausing at intermediate steps. However, plan to complete the entire upgrade within 60 days. This is critical to ensure that normal cluster automation processes are completed including those associated with certificate rotation.
+* You must be running at least 4.8.14 before starting the EUS-to-EUS upgrade procedure. If you do not meet this minimum requirement, upgrade to a later 4.8.z before attempting the EUS-to-EUS upgrade.
```

Comment:
Another conflict with what's above, where the version had to be later than 4.8.14. (unless I'm confusing different things.)